include/nbl/builtin/hlsl/glsl_compat/core.hlsl

```diff
@@ -224,9 +223,29 @@ struct bitfieldInsert
 } //namespace impl
 
 template<typename T>
-T bitfieldInsert( T base, T insert, uint32_t offset, uint32_t count )
+T bitfieldInsert( T base, T insert, uint32_t offset, uint32_t bits )
+{
+    return impl::bitfieldInsert<T>::__call(base, insert, offset, bits);
+}
+
+namespace impl 
+{
+
+template<typename T>
+struct bitfieldReverse
+{
+    static T __call( T base )
+    {
+        return spirv::bitFieldReverse<T>( base );
+    }
+};
+
+} //namespace impl
```

Comment:
at this point, what does going through a struct for insert and reverse give you?

Reply:
I was replicating what bitfield extract was doing but now that you called it out I realize it's only doing that to have partial specializations. Fixing now